packages/xrpl/test/models/permissionedDomainSet.test.ts

```
@@ -0,0 +1,51 @@
+    assert.doesNotThrow(() => validate(tx))
+  })
+
+  it(`throws with invalid field DomainID`, function () {
+    tx.DomainID = 1234 // DomainID is expected to be a string
+    const errorMessage = 'PermissionedDomainSet: invalid field DomainID'
+    assert.throws(() => validatePermissionedDomainSet(tx), ValidationError, errorMessage)
+    assert.throws(() => validate(tx), ValidationError, errorMessage)
+  })
+
+  it(`throws w/ missing field AcceptedCredentials`, function () {
+    delete tx.AcceptedCredentials
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)